crates/bevy_feathers/src/controls/button.rs

```diff
@@ -183,6 +183,11 @@ fn set_button_colors(
         (ButtonVariant::Primary, false) => tokens::BUTTON_PRIMARY_TEXT,
     };
 
+    let cursor_shape = match disabled {
+        true => bevy_window::SystemCursorIcon::NotAllowed,
+        false => bevy_window::SystemCursorIcon::Pointer,
+    };
```

Comment:
It's not a problem to match on a bool I guess, but it's not idiomatic and looks a bit odd:
```suggestion
    let cursor_shape = if disabled {
        bevy_window::SystemCursorIcon::NotAllowed
    } else {
        bevy_window::SystemCursorIcon::Pointer
    };
```